src/lib/get-verified-fetch.ts

```diff
@@ -65,6 +68,7 @@ export async function getVerifiedFetch (config: ConfigDb, logger: ComponentLogge
       libp2p,
       routers,
       blockBrokers,
+      hashers,
```

Comment:
FYI @2color we may want to switch this to `loadHasher` https://github.com/ipfs/helia/blob/aecac3d92cbd22a7331afee8e6f87ef31a9f7d95/packages/utils/src/index.ts#L75 at some point